dashboard/db/migrate/20170331235243_create_experiments.rb

```diff
@@ -0,0 +1,16 @@
+class CreateExperiments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :experiments do |t|
+      t.string :name
+      t.string :type
+      t.datetime :start_time
+      t.datetime :end_time
+      t.integer :section_id
+      t.integer :percentage
+      t.datetime :earliest_section_start
+      t.datetime :latest_section_start
```

Comment:
Added a couple `null: false`s.

The datetimes are all optional, but that leads to a bunch of queries like `start_time IS NULL OR start_time < ?`. Would it be too much of a hack to give the start times a default in the past, and the end times a default far in the future, so that all queries can be simplified to `start_time < ?`?